### PR TITLE
Update sql_sqlite_dialect.rst to include single quote example

### PR DESCRIPTION
**Pull request by jidanni**
Also folded monster long line.

(Single quotes have actually been part of the shell for 50 years.)

### CODE REVIEW

File: doc/source/user/sql_sqlite_dialect.rst [hunk 1]
```diff
@@ -115,6 +116,12 @@ between double quotes, the internal double quotes must be escaped with \\
 
     ogrinfo p.shp -sql "SELECT geometry \"FROM\", \"3D\" FROM p"
 
+If single quotes are available in the shell, one can also use just
+
```
Comment (jratike80) [inline doc/source/user/sql_sqlite_dialect.rst:120]: Single quotes are available in the Windows shell but they can be used only on Linux (and Mac?). I am not sure if it is good to recommend using a syntax that does not work on all platforms.

There needs to be a comma after "geometry".

**Comment by jidanni** [comment]
Well please mention that there, else, for us who have never used the other systems you mention, it looks like the author is just making things 50 times harder for themselves, because a vital page was creased in their K&R Book.

**Comment by jratike80** [comment]
Rather write it into a supporting document "Good to know about Linux vs Windows" because the use of single vs double quotes is something that does not only affect the usage of SQLite SQL dialect.

**Comment by jidanni** [comment]
I'm sorry I can't make sense of the diffs here on GitHub. All I know is the current version on the website seems like grandparents paying for their new house with a stack of one dollar bills. So needs about three words added acknowledging that simpler solutions are available on some systems. Thanks. No need to even mention what they are though.

**Comment by jratike80** [comment]
Sorry, I don't understand your last comment. 

 Because you aim at clarity, then how about changing "If single quotes are available in the shell" into  "Linux and Mac users can use single outer quotes but that syntax does not work on Windows"

File: doc/source/user/sql_sqlite_dialect.rst [hunk 2]
```diff
@@ -115,6 +116,12 @@ between double quotes, the internal double quotes must be escaped with \\
...
+.. code-block:: shell
+
+    ogrinfo p.shp -sql 'SELECT geometry "FROM", "3D" FROM p'
+
```
Comment (rouault) [inline doc/source/user/sql_sqlite_dialect.rst:124]: Dropping that change
```suggestion
```

**Comment by rouault** [comment]
> Rather write it into a supporting document "Good to know about Linux vs Windows" because the use of single vs double quotes is something that does not only affect the usage of SQLite SQL dialect.

yes, that would be more appropriate in a more generic place, but not sure where it is belong to. Where do we stop in our explanations of what is not GDAL specific... ?

**Pull request merged**